integr8ly/tutorial-web-app
# INTLY-568: Change lists to PF4 data lists

## Motivation
INTLY-568 Add data lists component to Integreatly PF4 POC

## What
Changed lists in the applications list to use PF4 data list, fixed styles and tests. Also updated the 3 patternfly dependencies to their latest version.

## Why
Part of the initiative to replace all PF3 components to PF4.

## Verification Steps
1. Go to the main Integreatly dashboard home page.
2. Verify that the Applications list on the right appears and functions as expected.

## Checklist:

## Progress

## Additional Notes
Screen shot of PF4 applications list:
![applications_list](https://user-images.githubusercontent.com/39063664/53046309-c1d3d900-345d-11e9-92e7-6ddd57218160.png)
